utils/searchFilters.ts

```diff
@@ -37,6 +37,14 @@ export function parsedQueryToSearchFilters(urlQuery: ParsedUrlQuery) {
         if (float) filters[key] = float;
         break;
 
+      case "bbox":
+        const list:any = urlQuery[key];
+        if (list && list.length) {
```

Comment:
I think it'd be safer to narrow the type with `Array.isArray(urlQuery[key])` instead of using `any`